build.gradle.kts

```diff
@@ -172,8 +172,8 @@ allprojects {
       param.environment("PROJECT_VERSION", project.version)
 
       // Gravitino CI Docker image
-      param.environment("GRAVITINO_CI_HIVE_DOCKER_IMAGE", "apache/gravitino-ci:hive-0.1.18")
-      param.environment("GRAVITINO_CI_KERBEROS_HIVE_DOCKER_IMAGE", "apache/gravitino-ci:kerberos-hive-0.1.5")
+      param.environment("GRAVITINO_CI_HIVE_DOCKER_IMAGE", "unknowntpo/gravitino-hive-ci-test:hive-test")
+      param.environment("GRAVITINO_CI_KERBEROS_HIVE_DOCKER_IMAGE", "unknowntpo/gravitino-kerberos-hive-ci-test:kerberos-hive-test")
```

Comment:
@unknowntpo Waiting Github Action passed, Please don't forget modify this env values.

Reply:
ok, I've added a TODO list in description as reminder.